lib/teamsnap.rb

```diff
@@ -137,13 +138,40 @@ def init(opts = {})
       collection = TeamSnap.run_init(:get, "/", {})
 
       classes = []
-      client.in_parallel do
-        classes = collection
-          .fetch(:links)
-          .map { |link| classify_rel(link) }
+
+      schema = collection
+        .fetch(:links) {{}}
+        .detect { |link| link[:rel] == "schemas" }
+
+      if schema && schema[:href]
```

Comment:
perhaps put an `|| {}` when setting the `schema` variable? that way you can check `schema[:href]` only.